tests/remote_persistence_test.rs

```diff
@@ -0,0 +1,69 @@
+#[path = "print_events_handler/mod.rs"]
+mod print_events_handler;
+#[path = "../eel/tests/setup/mod.rs"]
+mod setup;
+mod setup_3l;
+
+#[cfg(feature = "nigiri")]
+mod node_info_test {
+    use crate::setup::nigiri;
+    use crate::setup::nigiri::NodeInstance;
+    use crate::setup_3l::NodeHandle;
+    use crate::try_cmd_repeatedly;
+
+    use bitcoin::hashes::hex::ToHex;
+    use serial_test::file_serial;
+    use std::process::{Command, Output};
+    use std::thread::sleep;
+    use std::time::Duration;
+
+    const HALF_SEC: Duration = Duration::from_millis(500);
+    const N_RETRIES: u8 = 10;
+
+    #[test]
+    #[file_serial(key, "/tmp/3l-int-tests-lock")]
+    fn test_remote_persistence() {
+        nigiri::setup_environment_with_lsp();
+
+        let node_handle = NodeHandle::new();
+
```

Comment:
We should think about how to avoid bloating the backend CSP service with test channel states. Maybe use a specific tag on the custom fields and use that to clean the db? Another option is to mock `mole` such that we still test the integration between `eel` and `mole` without needing the backend.

Reply:
I don't necessarily agree that it is `3L`'s responsibility to keep the backend database clean. As long as we use the `DEV` or `STAGING` environment this should be fine imo. These environments are meant to be spammed with non-sense and clean ups should be in there own responsibility.

There is a point to be made about mocking the backend though. Independence. Our integration tests would still work when the backend is down or broken. That can be helpful, particularly with a flaky kind of backend like the one we're dealing with atm.

The drawback of course, is that it's not *really* an integration test anymore because it doesn't test the backend and 3L's integration with it anymore.

Reply:
I see your point, but still think we should make it easier for the clean-ups to not remove channel states from test wallets we want to be a bit longer lived. These very short-lived wallets created in int tests could have appended to the 3L version some code that distinguishes them. WDYT?

> The drawback of course, is that it's not really an integration test anymore because it doesn't test the backend and 3L's integration with it anymore.

It would still be an integration test testing the integration of `eel` and `mole`. Just not the integration with the backend :)

Reply:
Let's deal with this in another PR